setup.py

```diff
@@ -21,7 +21,11 @@
             "pytest==2.9.1",
         ]
     },
-    dependency_links=[],
+    entry_points={
```

Comment:
that's nice. in that vein we might rename `base` to `declarative_base` or similar?